app/models/plugin.py

```
@@ -0,0 +1,23 @@
+from pydantic import BaseModel
+from typing import Optional, Any
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)